script.js

```diff
@@ -0,0 +1,95 @@
+const hamberger = document.querySelector('.hamberger');
+const close = document.querySelector('.close');
+const body = document.querySelector('body');
+const menu = document.querySelector('.menu');
+const bodyOverlay = document.querySelector('.body-overlay');
+
+hamberger.addEventListener('click', () => {
+  close.classList.remove('hidden');
+  bodyOverlay.classList.remove('hidden');
+  menu.classList.add('hamberger-menu');
+  hamberger.classList.add('hidden');
+  body.classList.add('non-scroll');
+});
+
+close.addEventListener('click', () => {
+  close.classList.add('hidden');
+  bodyOverlay.classList.add('hidden');
+  menu.classList.remove('hamberger-menu');
+  hamberger.classList.remove('hidden');
+  body.classList.remove('non-scroll');
+});
+
+const speakers = [
+  {
+    name: 'Saeedeskandari',
+    title: 'CEO',
+    imgFeture: 'https://userphotos2.teacheron.com/1530294-07160.jpg',
+    description: 'I am fully expert in using AutoCAD, Revit and Excel, Because of years experience use of them in many projects.',
+  },
+  {
+    name: 'Kayla Snow',
+    title: 'AutoCAD and Revit Architect',
+    imgFeture: 'https://userphotos2.teacheron.com/1529707-03285.jpg',
+    description: 'career drafter teaching Revit and 2D AutoCAD skills',
+  },
+  {
+    name: 'Takeru Kondo',
+    title: 'BIM Specialist',
+    imgFeture: 'https://userphotos2.teacheron.com/1513636-15272.jpg',
+    description: 'I am licensed architect with over 10 years of experience in commercial architecture.',
+  },
+  {
+    name: 'Rizwan Razzak Ali',
+    title: 'CAD/CFD/FEA Expert',
+    imgFeture: 'https://userphotos2.teacheron.com/1511945-15254.jpeg',
+    description: 'I achieved many certificates from Dassault Systems, AutoDesk, and Bentley USA for excellent Design skills',
+  },
+  {
+    name: 'Emmanuel Kuchocha',
```

Comment:
- [ ] What do you think to make list of the same number of speakers like the ones from the template to match the expected design.
- [ ] Also their design should be close to the ones from the template, like the grid-background image of the profile picture for each user.

**View on Desktop version**
| Template design | Your Design |
| --------------------- | --------------------- |
| ![Screenshot from 2023-01-05 20-34-10](https://user-images.githubusercontent.com/83042742/210855270-e39f3d6c-23f8-4459-bc20-ea40281a4fed.png) | ![Screenshot from 2023-01-05 20-35-07](https://user-images.githubusercontent.com/83042742/210855421-f8d055e8-e45b-4a02-ad88-e5f82dc8cc67.png) |

**View on some Mobile version widths size**
![Screenshot from 2023-01-05 20-39-17](https://user-images.githubusercontent.com/83042742/210855885-ac08ba7a-ab31-41c7-8aa6-d6ed3339a29c.png)